julia_src/Manifest.toml

```diff
@@ -917,9 +917,11 @@ uuid = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 
 [[deps.REopt]]
 deps = ["ArchGDAL", "CSV", "CoolProp", "DataFrames", "Dates", "DelimitedFiles", "HTTP", "JLD", "JSON", "JuMP", "LinDistFlow", "LinearAlgebra", "Logging", "MathOptInterface", "Requires", "Roots", "Statistics", "TestEnv"]
-git-tree-sha1 = "d2bbcbb8344f19ed87ee8cb6a196d4ed8415255b"
+git-tree-sha1 = "28bbe232da58b84194f3c1b4f0d60d577f24ba2c"
+repo-rev = "fix-ghp-storage-dispatch"
+repo-url = "https://github.com/NREL/REopt.jl.git"
 uuid = "d36ad4e8-d74a-4f7a-ace1-eaea049febf6"
-version = "0.45.0"
+version = "0.46.0"
```

Comment:
Just a note to update this to the 46.1 eventually

Reply:
Thanks for the reminder! Addressed here: https://github.com/NREL/REopt_API/commit/ae8fceac0a30a8552d465260066a59d28f2edd00